kakao/src/main/kotlin/com/agoda/kakao/Matchers.kt

```diff
@@ -148,6 +149,18 @@ class IndexMatcher(private val matcher: Matcher<View>, private val index: Int) :
 }
 
 /**
+ * Matcher of value rating of given matcher
+ * @param value of rating that matched the view which is rating bar
+ */
+class RatingBarMatcher(private val value: Float) : BoundedMatcher<View, RatingBar>(RatingBar::class.java) {
+    override fun matchesSafely(view: RatingBar?) = view?.let { view.rating == value } ?: false
```

Comment:
Could you change from `view` to `it` inside `let` block?

Reply:
yes. I can. Or can I use run ?